frontend/packages/noobaa-storage-plugin/src/constants/data-consumption.ts

```diff
@@ -0,0 +1,17 @@
+export const ACCOUNTS = 'Accounts';
+export const PROVIDERS = 'Providers';
+export const BY_IOPS = 'I/O Operations';
+export const BY_LOGICAL_USAGE = 'Logical Used Capacity';
+export const BY_PHYSICAL_VS_LOGICAL_USAGE = 'Physical Vs Logical Usage';
+export const BY_EGRESS = 'Egress';
+export const PROJECTS = 'Projects';
+export const BUCKET_CLASS = 'Bucket Class';
+export const OCS_NS = 'openshift-storage';
+export const NB_PROVISIONER = 'noobaa.io/obc';
```

Comment:
This should be not be in data-consumption, rather in a generic file e.g `common.ts`